flocker/acceptance/endtoend/test_dockerplugin.py

```diff
@@ -20,6 +25,36 @@ class DockerPluginTests(TestCase):
     """
     Tests for the Docker plugin.
     """
+    def restart_docker(self, address):
+        """
+        Restart the Docker daemon on the specified address.
+
+        :param bytes address: The public IP of the node on which Docker will
+            be restarted.
+        """
+        distro = []
+        get_distro = run_ssh(
+            reactor, b"root", address, ["python", "-m", "platform"],
+            handle_stdout=distro.append)
+        get_distro.addCallback(lambda _: distro[0].lower())
+
+        def restart_docker(distribution):
+            if 'ubuntu' in distribution:
+                command = ["service", "docker", "restart"]
+            else:
+                command = ["systemctl", "restart", "docker"]
+            d = run_ssh(reactor, b"root", address, command)
+
+            def handle_error(_):
+                self.fail(
+                    "Restarting Docker failed. See logs for process output.")
```

Comment:
:question: Isn't there anything useful in the supplied `Failure` (`_`) which would make debugging easier?